rotkehlchen/chain/evm/decoding/hop/decoder.py

```diff
@@ -333,13 +369,18 @@ def _decode_add_liquidity(self, context: DecoderContext) -> DecodingOutput:
             elif (
                 event.event_type == HistoryEventType.RECEIVE and
                 event.event_subtype == HistoryEventSubType.NONE and
-                event.location_label == user_address
+                event.location_label == user_address and
+                event.asset.is_evm_token()
```

Comment:
why this check?

Reply:
this was because we `resolve_to_evm_token()` below, so to ensure it doesn't raise any error. But I'm removing it now and try/catching the error